air/src/execution_step/execution_context/instruction_error/error_descriptor.rs

```diff
@@ -34,6 +36,11 @@ impl ErrorDescriptor {
     ) {
         use super::get_instruction_error_from_exec_error;
 
+        // Returns early if there is an error to bubble up.
+        if !self.error_can_be_set {
```

Comment:
let's write these two conditions in one if block

Reply:
Fixed